docs/source/getting_started/examples/sglang-integration/hicache-integration-v1.md

```diff
@@ -0,0 +1,253 @@
+# SGLang HiCache with Mooncake Backend
```

Comment:
Should we include a link or reference to sglang?

Reply:
Sure. I added the link of sglang to the **About SGLang HiCache** section, just similar to Mooncake.